.env.example

```diff
@@ -45,4 +45,8 @@ REDIS_CLUSTER_IP_TEST=localhost
 
 # Data Source URL for Sentry
 SENTRY_DSN=
-SENTRY_DSN_TEST=localhost
\ No newline at end of file
+SENTRY_DSN_TEST=localhost
+
+# Threshold for triggering a Radar Heartbeat Alert
+RADAR_THRESHOLD_MILLIS=1200000
+DOOR_THRESHOLD_MILLIS=1200000
```

Comment:
You decided against have `_TEST` versions of these?

Reply:
I didn't explicitly. That said, we don't run heartbeat code in test mode anyways.